accern_xyme/util.py

```diff
@@ -171,6 +173,10 @@ def interpret_ctype(data: IO[bytes], ctype: str) -> ByteResponse:
         return json.load(data)
     if ctype == "application/parquet":
         return pd.read_parquet(data)
+    if ctype == "application/torch":
+        return torch.load(data)
```

Comment:
this requires torch to be installed -- can you move the imports here

Reply:
same with scipy -- if a library is not in the requirements -- we cannot import it globally
also, if it is not installed we return a BytesIO so please capture module or import errors